src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -2,17 +2,19 @@
 
 import java.util.Objects;
 
-public class LottoNumber implements Comparable<LottoNumber>{
+public class LottoNumber implements Comparable<LottoNumber> {
 
     private static final int MIN_LOTTO_NUMBER = 1;
     private static final int MAX_LOTTO_NUMBER = 45;
 
     private final int number;
+
     public LottoNumber(int number) {
         this.number = number;
+        validate();
```

Comment:
```suggestion
        validate();
        this.number = number;
```
유효성 체크를 먼저 한 후에 유효한 값이면 필드 초기화할 것을 추천